MangaDex issue Source Settings stuck at Loading broken access token

Loading a new device with official v8.9-rc28 and beta v8.0-rc3 build and installing this repo, then adding MangaDex leads to a string of pop up errors: 
[ERROR] [<timestamps>] [DUISection:reloadRows():140] JSError ReferenceError [PaperbackError(300)] JSError ReferenceError: Can’t find variable: Buffer -@ 
[ERROR] [<timestamps>] [DUISection:reloadRows():140] parseAccessToken@
[ERROR] [<timestamps>] [DUISection:reloadRows():140]  @:1991:33

Also affects existing device if you press the Reload button on the source controls

![IMG_1193](https://github.com/user-attachments/assets/a19008d1-6312-468d-a3c0-a2fa92b6fb7c)

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
   "dependencies": {
     "@paperback/toolchain": "^0.8.7",
     "@paperback/types": "^0.8.7",
+    "buffer": "^6.0.3",
     "cheerio": "^1.0.0",
     "html-entities": "^2.5.2"
   }

--- a/src/MangaDex/MangaDex.ts
+++ b/src/MangaDex/MangaDex.ts
@@ -64,7 +64,7 @@ export const MangaDexInfo: SourceInfo = {
     description: 'Extension that pulls manga from MangaDex',
     icon: 'icon.png',
     name: 'MangaDex',
-    version: '3.0.5',
+    version: '3.0.6',
     authorWebsite: 'https://github.com/nar1n',
     websiteBaseURL: MANGADEX_DOMAIN,
     contentRating: ContentRating.EVERYONE,

--- a/src/MangaDex/MangaDexSettings.ts
+++ b/src/MangaDex/MangaDexSettings.ts
@@ -7,7 +7,9 @@ import {
     MDRatings,
     MDImageQuality
 } from './MangaDexHelper'
-
+import {
+    Buffer
+} from 'buffer'
 
 export async function getLanguages(stateManager: SourceStateManager) {
     return (await stateManager.retrieve('languages') ?? MDLanguages.getDefault())

